prediction_market_agent/agents/crewai_subsequential_agent/benchmark.py

```diff
@@ -0,0 +1,157 @@
+                is_predictable=True,
+                outcome_prediction=OutcomePrediction(
+                    p_yes=answer.result.to_p_yes(),
+                    confidence=1.0,
+                    info_utility=None,
+                ),
+            )
+
+
+if __name__ == "__main__":
+    load_dotenv()
+    tomorrow_str = (utcnow() + timedelta(days=1)).strftime("%d %B %Y")
+
+    # Fetch example questions which our agents answered in the past.
+    questions = [
+        QuestionAndAnswer(
+            question="Will the stock price of Donald Trump's media company exceed $100 on 1 April 2024?",
+            result=Result.NO,
+            bet_correct=True
+        ),
+        QuestionAndAnswer(
+            question="Will Andy Murray return to professional tennis from his ankle injury on or before 31 March 2024?",
+            result=Result.NO,
+            bet_correct=True
+        ),
+        QuestionAndAnswer(
+            question="Will any legislation be signed by President Biden that could potentially lead to the ban of TikTok by 1 April 2024?",
+            result=Result.YES,
+            bet_correct=False
+        ),
+        QuestionAndAnswer(
+            question="Will the United States v. Apple case have a verdict by 1 April 2024?",
+            result=Result.NO,
+            bet_correct=True
+        ),
+        QuestionAndAnswer(
+            question="Will Microsoft Teams launch the announced Copilot AI features by 1 April 2024?",
+            result=Result.YES,
+            bet_correct=True
+        ),
+        QuestionAndAnswer(
+            question="Will the Francis Scott Key Bridge in Baltimore be fully rebuilt by 2 April 2024?",
+            result=Result.NO,
+            bet_correct=True
+        ),
+        QuestionAndAnswer(
+            question="Will iOS 18 break the iPhone's iconic app grid by 1 April 2024?",
+            result=Result.YES,
+            bet_correct=False
+        ),
+        QuestionAndAnswer(
+            question="Will a winner of the Mega Millions jackpot be announced by 26 March 2024?",
+            result=Result.YES,
+            bet_correct=False
+        ),
+    ]
+
+    benchmarker = Benchmarker(
+        markets=[build_market_from_question_without_validation(q.question) for q in questions][:1],
+        agents=[
+            CrewAIAgentSubquestionsBenchmark(
+                agent_name="subsequential_questions",
+                model="gpt-3.5-turbo-0125",
+                max_tries=3,
+                max_workers=1,
+            ),
+        ],
+    )
+    benchmarker.run_agents()
+    md = benchmarker.generate_markdown_report()
+
+    output = f"./subsequential_questions_agent_benchmark_report.{int(time.time())}.md"
+    with open(output, "w") as f:
+        print(f"Writing benchmark report to: {output}")
+        f.write(md)
+
+    # Check all predictions are correct, i.e. mean-squared-error == 0
+    metrics = benchmarker.compute_metrics()
+    assert metrics["MSE for `p_yes`"][0] == 0.0
```

Comment:
@evangriffiths Not sure if this line is indeed required for every run - the report is produced correctly. 
I copied this from your benchmark script.

Reply:
That can be removed as it won't be the case once you use markets from Polywrap/Manifold :)